cluster-autoscaler/go.mod

```diff
@@ -25,6 +25,14 @@ require (
 	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	google.golang.org/api v0.6.1-0.20190607001116-5213b8090861
+	google.golang.org/grpc-new v1.40.0
+	google.golang.org/protobuf-new v1.27.1
+	//github.com/golang/protobuf-new
```

Comment:
Not sure what this stuff is?

Reply:
I commented this because I directly copied over what I *think* are those versions from the master branch vendor file by looking at the go.sum there